radio/src/io/frsky_firmware_update.cpp

```diff
@@ -752,8 +757,9 @@ const char * FrskyChipFirmwareUpdate::flashFirmware(const char * filename, bool
   uint8_t extPwr = IS_EXTERNAL_MODULE_ON();
   EXTERNAL_MODULE_OFF();
 
+  uint8_t spuPwr = IS_SPORT_UPDATE_POWER_ON();
   SPORT_UPDATE_POWER_OFF();
-
+  
```

Comment:
Cosmetics

Reply:
sorry, not sure...naming of spuPwr or else?

Reply:
the space character